novawallet/Modules/Notifications/Management/NotificationsManagementInteractor.swift

```diff
@@ -75,9 +85,44 @@ final class NotificationsManagementInteractor: AnyProviderAutoCleaning {
                         chains: chains
                     )
                     self.localSettings = .defined(defaultSettings)
-                    self.presenter?.didReceive(settings: defaultSettings)
+                    self.provideSettings()
+                }
+            }
+        }
+    }
+
+    func fetchWallets() {
+        let fetchWalletsOperation = walletsRepository.fetchAllOperation(with: .init())
```

Comment:
As we have now a logic (in a separate PR) that syncs locally available wallets with remote do we still need to have this here? I don't think so

Reply:
You are right, now it is not actual